### PR TITLE
Transformers revisit

### PR DESCRIPTION
**Pull request by niwinz**

### CODE REVIEW

File: src/cats/protocols.cljc [hunk 1]
```diff
@@ -24,23 +24,19 @@
 ;; THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 (ns cats.protocols
-  "Abstractions of Category Theory over that
-  cats library is build.
+  "A collection of protocols on that cats
+  abstractions is built up on.
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: This sentence doesn't make sense. It should be:

> A collection of protocols upon which the cats abstractions are built.

File: src/cats/protocols.cljc [hunk 2]
```diff
@@ -24,23 +24,19 @@
...
 
-  Functions of this namespace are not indented
-  to be used directly. Is a private api but exposes
-  as public for documentation purposes.")
+  NOTE: Functions of this namespace are not indented
+  to be used directly. It is considered internal api.")
 
 (defprotocol ContextClass
-  "Is a marker protocol for context classes."
+  "A marker protocol for identify the valid context types."
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: s/identify/identifying/

File: src/cats/protocols.cljc [hunk 3]
```diff
@@ -24,23 +24,19 @@
...
   (-get-level [_] "Get a context priority level."))
 
 (defprotocol Context
-  "Abstraction that establish a membership of types
-  with one concrete monad.
-
-  This is a way that cats establishes the relation
-  between a type and the monad that that should play.
+  "Abstraction that establish a membership between the
+  context and the concrete type.
```
Comment (yurrriq) [inline src/cats/protocols.cljc]: This is also oddly worded.. Perhaps:

> Abstraction that establishes a concrete type as a member of a context.

**Comment by yurrriq** [comment]
I don't mean for the grammar criticisms to come off the wrong way. If you want me to stop, please don't hesitate to tell me.

I'm planning to do a big docs overhaul (like I did in `cats.core`) when I get time.

**Comment by niwinz** [comment]
In contrary. Your fixes, improvements and suggestions are more that than welcome. Thanks!

**Comment by yurrriq** [comment]
:)

File: test/cats/labs/reader_spec.cljc [hunk 4]
```diff
@@ -29,9 +29,9 @@
   (t/is (= 42 (reader/run-reader (m/fmap inc reader/ask) 41))))
 
 
-(def maybe-reader-t (reader/reader-transformer maybe/context))
+(def maybe-reader-t (reader/reader-t maybe/context))
 
-(t/deftest reader-transformerformer-tests
+(t/deftest reader-tformer-tests
```
Comment (dialelo) [inline test/cats/labs/reader_spec.cljc]: we should name it `reader-t-tests` for consistency.

**Comment by niwinz** [comment]
Fixed!

**Pull request merged**